README.md

```diff
@@ -39,6 +39,7 @@ ESLint is a tool for identifying and reporting on patterns found in ECMAScript/J
 10. [License](#license)
 11. [Team](#team)
 12. [Sponsors](#sponsors)
+<!-- markdownlint-disable-next-line MD051 -->
 13. [Technology Sponsors](#technology-sponsors)
```

Comment:
```suggestion
13. [Technology Sponsors](#technology-sponsors) <!-- markdownlint-disable-line MD051 -->
```

To avoid a blank line.

![image](https://github.com/eslint/eslint/assets/44349756/37fa5aba-96c6-4158-89dc-9dadc2d5ddea)